config/.jvm/src/test/scala/bloop/config/TestPlatform.scala

```diff
@@ -14,7 +13,7 @@ object TestPlatform {
         val isr = new InputStreamReader(stream)
         try {
           val reader = new BufferedReader(isr)
-          reader.lines().collect(Collectors.joining(lineSeparator))
+          reader.lines().collect(Collectors.joining(" "))
```

Comment:
This is actually used in a test and it's not needed for a json to have `\n`. Also, `bloop.io.Environment.lineSeparator` is not available in config module, but that was never checked, since tests are not run on config. It came up here, because I added `test -> test` dependency to have simple base class for different config tests. I could add a separate module, but the way it's currently done would mean that I would have to add 3 new modules.

I will tidy it up further down the line.